addons/website_forum/models/website.py

```diff
@@ -50,7 +50,8 @@ def _update_forum_count(self):
         websites (to update globally counters, notably when a new forum is
         created). """
         websites = self if self else self.search([])
+        forums_all = self.env['forum.forum'].search([])
         for website in websites:
-            website.forum_count = self.env['forum.forum'].sudo().search_count(
+            website.forum_count = len(forums_all.filtered_domain(
```

Comment:
Independently from our upcoming talk :black_flag: :smile: , this could easily be inlined
```suggestion
            website.forum_count = len(forums_all.filtered_domain(website.website_domain()))
```